forms-flow-web/src/components/Form/List.scss

```diff
@@ -112,5 +114,9 @@
     .flex-item-right,
     .flex-item-left {
         flex: 100%;
+        justify-content: flex-start;
+    }
+    .col-md-6 {
```

Comment:
please avoid overwrite  the bootstrap class

Reply:
good catch